### ISSUE TITLE
Accessibility: DataGridView cells in a ReadOnly DGV are reported as editable and Value.IsReadOnly is false in Inspect

### ISSUE THREAD

**Issue by Olina-Zhang** [dotnet/winforms#1578]
* .NET Core Version: .Net 3.0 SDK Preview8 013656 build
* Have you experienced this same bug with .NET Framework?: Yes

**Repro Steps:**
1. Create a Winforms .Net Core application

2. Add DataGridView control, set ReadOnly property to true

3. Add columns and rows

    public Form1()
        {
            InitializeComponent();
            this.dataGridView1.ReadOnly = true;
            this.dataGridView1.Rows.Insert(0, new object[3] { "aa", "bb", "cc" });
            this.dataGridView1.Rows.Insert(0, new object[3] { "aa1", "bb1", "cc1" });
        }

4. Run app and navigate to a cell in narrator or check the Value.IsReadOnly property in Inspect 
[WindowsFormsApp5.zip](https://github.com/dotnet/winforms/files/3475785/WindowsFormsApp5.zip)

**Actual behavior:**
In Narrator, it reads the cell with editing state. And in Inspect, its Value.IsReadyOnly is false. Please see following screenshot:
![image](https://user-images.githubusercontent.com/26474449/62604294-9f6b3600-b8ac-11e9-8254-745fc1fec9c9.png)

**Expected behavior:**
In Narrator, it should announce the cell with ReadOnly state, and In Inspect, Value.IsReadOnly property should be true when focusing on cell.

**Comment by Jessie-Zhang01** [comment]
Verified this bug in .Net Core SDK 3.0.100-Preview 9-13959 build from release branch. This bug is fixed. Please see following screenshot:
![image](https://user-images.githubusercontent.com/45864649/63500792-20593d00-c4fd-11e9-995b-dfa2854d9719.png)

**Comment by RussKie** [comment]
Thank you @Jessie-Zhang01
